src/xo-app/vm/tab-disks.js

```diff
@@ -189,7 +278,6 @@ class AttachDisk extends Component {
   _addVdi = () => {
     const { vm, vbds, onClose = noop } = this.props
     const { bootable, readOnly, vdi } = this.state
-
```

Comment:
Why?

Reply:
Because, the following line contains another initialization with const. :)